src/assets/stylesheets/index.scss

```diff
@@ -23,3 +25,65 @@ code {
 #app {
   min-height: 100dvh;
 }
+.--light {
+  --editor-color-layer-1: #{$rpf-teal-100};
+  --editor-color-layer-2: #{$rpf-white};
+  --editor-color-layer-3: #{$rpf-white};
+  --editor-color-outline: #{$rpf-grey-150};
+  --editor-color-theme: #{$rpf-teal-800};
+  --editor-color-theme-secondary: #{$rpf-teal-900};
+  --editor-color-theme-tertiary: #{$rpf-teal-100};
+  --editor-color-text: #{$rpf-text-primary};
+  --editor-color-text-secondary: #{$rpf-text-secondary};
+}
+
+.--dark {
+  --editor-color-layer-1: #{$rpf-grey-850};
+  --editor-color-layer-2: #{$rpf-grey-800};
+  --editor-color-layer-3: #{$rpf-grey-700};
+  --editor-color-outline: #{$rpf-grey-600};
+  --editor-color-theme: #{$rpf-teal-400};
+  --editor-color-theme-secondary: #{$rpf-teal-400};
+  --editor-color-theme-tertiary: #{$rpf-teal-900};
+  --editor-color-text: #{$rpf-white};
+  --editor-color-text-secondary: #{$rpf-text-secondary-darkmode};
```

Comment:
Are all these vars being defaulted for the web component anywhere? The web component test page on this branch looks a bit broken 😕 https://staging-editor.raspberrypi.org/branches/issues_746_Web_component_theming/web-component.html